scaffolds/Lite/src/router.jsx

```diff
@@ -1,8 +1,8 @@
 import { HashRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
 import React, { Suspense } from 'react';
 import path from 'path';
-import routes from '@/config/routes';
-import PageLoading from '@/components/PageLoading';
+import routes from 'config/routes';
+import PageLoading from 'components/PageLoading';
```

Comment:
Looks like have not serviceWorker.js file?

Reply:
Yes I didn't import that file from a CRA app, do we want that?

Reply:
serviceWorker.js looks like a symbol of CRA.